src/ts/components/Match.tsx

```diff
@@ -32,6 +38,14 @@ class Match<TMatch extends IMatch> extends Component<IMatchProps<TMatch>> {
                 matchId={matchId}
                 suggestions={suggestionsToRender}
               />
+              <div className="MatchWidget__feedbackLink">
+                <a 
+                  target="_blank"
+                  href={getFeedbackLink(feedbackInfo)}
+                >
+                  Something's not right? Tell us!
```

Comment:
`nit` Not sure if this reads very well - maybe it could change to 'Something not right? Tell us!' Or do we need to be even more specific that something's not right with **this particular** match?

Reply:
This text is from the UI design.  I'm not changing it.